nixnet/errors.py

```diff
@@ -74,16 +75,17 @@ def __init__(self, message, error_code):
     def error_code(self):
         """Error code reported by NI-XNET.
 
-        int: Specifies the NI-XNET error code.
+        Returns:
```

Comment:
From the example I found, looks like we are both wrong on how to document a property